src/covid19model/models/models.py

```diff
@@ -163,8 +163,11 @@ def integrate(t, S, E, I, A, M, C, C_icurec, ICU, R, D, SQ, EQ, IQ, AQ, MQ, RQ,
         dMQ = theta_M*psi_PP*M + (m/omega)*IQ - ((1-h)/dm)*MQ - (h/dhospital)*MQ
         dRQ = theta_R*psi_FP*R - RQ/dq
 
+        H_in=(M+MQ)*(h/dhospital)
+        H_out =  C*(1/dc) + (m0/dICU)*ICU + C_icurec*(1/dICUrec)
+
         return (dS, dE, dI, dA, dM, dC, dC_icurec,
-                dICUstar, dR, dD, dSQ, dEQ, dIQ, dAQ, dMQ, dRQ)
+                dICUstar, dR, dD, dSQ, dEQ, dIQ, dAQ, dMQ, dRQ, H_in, H_out)
```

Comment:
I believe you are sending `H_in` and `H_out` to the integrator. So I think your model is outputting the cumulative sum of the ingoing and outgoing 'flows' of the hospital instead of the daily ingoing and outgoing flows ( = rate of change). I suggest the following change to return the true rates,
```
 dH_in = (M+MQ)*(h/dhospital) - H_in
 dH_out =  C*(1/dc) + (m0/dICU)*ICU + C_icurec*(1/dICUrec) - H_out
```
As a numeric example consider,

> At `t = 0` you specified an `H_in` of 0 patients (initial condition). Between `t = 0` and `t = 1`, 5 patients show up to the hospital,
> - `H_in, 0 = 0`
> - `dH_in, 0 = 5`

> At `t = 1`, the new `H_in` is calculated as the sum of the previous state `H_in,0` and the rate of change `dH_in,0`. Between `t = 1` and `t = 2`, 10 patients show up to the hospital,
> - `H_in,1 = H_in,0 + dH_in,0 = 5`
> - `dH_in, 1 = 10`

> Then, at `t = 2`, the new `H_in` is calculated once more as the sum of the previous state and the rate of change,
> - `H_in,2 = H_in,1 + dH_in,1 = 5 + 10 = 15`

Summarizing,
> `H_in, 0 = 0` (initial condition)
> `H_in,1 = 5`
> `H_in,2 = 15`

If you repeat the above with the suggested modification you will find,
> `H_in, 0 = 0` (initial condition)
> `H_in,1 = 5`
> `H_in,2 = 10`

Reply:
I accounted for this in the calibration (by taking the diff of H_in). But it would indeed be better to output the rate of change directly instead of the cumulative numbers! Then I'll change that in the calibration.
Could you add the dH_in and dH_out in the file where it should go exactly? Thanks!

Reply:
Never mind, I see where you mean